zerver/migrations/0268_add_userpresence_realm_timestamp_index.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-02-08 20:34
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('zerver', '0267_backfill_userpresence_realm_id'),
+    ]
+
+    operations = [
+        migrations.AlterIndexTogether(
+            name='userpresence',
+            index_together=set([('realm', 'timestamp')]),
```

Comment:
I think we can add the `realm not null` index to this migration rather than adding a new one.